### ISSUE TITLE
Legacy View Engine Deprecation Warning

### ISSUE THREAD

**Issue by ruizmarc** [MurhafSousli/ngx-gallery#419]
#### What is the expected behavior?

We expect the library to be distributed with Ivy. 

#### What is the current behavior?

When compilation happens we get this message: 
```
Processing legacy "View Engine" libraries:
ng-gallery [es2015/esm2015] (git://github.com/murhafsousli/ngx-gallery.git)
ng-gallery/lightbox [es2015/esm2015] (git://github.com/murhafsousli/ngx-gallery.git)
Encourage the library authors to publish an Ivy distribution.
```

#### What is the use-case or motivation for changing an existing behavior?

Legacy view engine will be removed in future versions. 

#### Which versions are you using for the following packages?

Angular: 13.1.0
Angular CDK: 13.1.0
Angular CLI: 13.1.0
Typescript: 4.4.4
Gallery: 5.1.1

#### Is there anything else we should know?

In order to distribute under Ivy, library should use Angular 13, enable Ivy and add in tsconfig:
```
{
...
  "angularCompilerOptions": {
    "compilationMode": "partial"
  }
...
}
```

**Comment by MurhafSousli** [comment]
Fixed in v6.0.0

**Comment by ruizmarc** [comment]
Thanks for being so quick! ❤️
